src/OpenTelemetry.Instrumentation.AspNetCore/CHANGELOG.md

```diff
@@ -2,13 +2,31 @@
 
 ## Unreleased
 
-* Updated Semantic Conventions to [v1.21.0](https://github.com/open-telemetry/semantic-conventions/blob/v1.21.0/docs/http/http-spans.md).
-  This library can emit either old, new, or both attributes.
-  Users can control which attributes are emitted by setting the environment
-  variable `OTEL_SEMCONV_STABILITY_OPT_IN`.
-  ([#4537](https://github.com/open-telemetry/opentelemetry-dotnet/pull/4537))
-  ([#4606](https://github.com/open-telemetry/opentelemetry-dotnet/pull/4606))
-  ([#4660](https://github.com/open-telemetry/opentelemetry-dotnet/pull/4660))
+* The new HTTP and network semantic conventions can be opted in to by setting
+  the `OTEL_SEMCONV_STABILITY_OPT_IN` environment variable. This environment
+  variable supports the following values:
+  * `http` - emit the new, stable HTTP and networking attributes, and stop
```

Comment:
```suggestion
  * `http` - emit the new, frozen (proposed for stable) HTTP and networking attributes, and stop
```

Reply:
Consider updating the Java release notes 😄 https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/tag/v1.27.0